gradle/verification-metadata.xml

```diff
@@ -229,6 +237,14 @@
             <sha256 value="3a2837580ca76af36af7dfe2dc1d334fc6ff5f26d10f24899db7770ad7cfd9fe" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apiguardian" name="apiguardian-api" version="1.1.2">
+         <artifact name="apiguardian-api-1.1.2.jar">
+            <sha256 value="b509448ac506d607319f182537f0b35d71007582ec741832a1f111e5b5b70b38" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="apiguardian-api-1.1.2.module">
+            <sha256 value="e08028131375b357d1d28734e9a4fb4216da84b240641cb3ef7e7c7d628223fc" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
```

Comment:
Is `apiguardian` actually required in order to support JUnit 5? If not, this should be reverted. If so, that's another third-party dependency we would need to get approval for.

Reply:
It's an annotation used within JUnit code, it might be possible to block this one, I will see how to do this in Gradle syntax.

This is the whole transitive tree it pulls in:
![image](https://user-images.githubusercontent.com/995917/134737975-8cc2aca3-6b58-488d-b9b5-b6c4906d653f.png)

Reply:
Blocked this one without ill effects, tests still run.